packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -49,15 +49,16 @@ export class HbarSpendingPlanConfigService {
   private readonly TTL: number = -1;
 
   /**
-   * The name of the spending plans configuration file. Defaults to `spendingPlansConfig.json`.
+   * The environment variable that contains the spending plans configuration.
+   * It can be either a file path or a JSON string.
    *
    * @type {string}
    * @private
    */
-  // @ts-ignore
-  private readonly SPENDING_PLANS_CONFIG_FILE: string =
-    ConfigService.get('HBAR_SPENDING_PLANS_CONFIG_FILE') || 'spendingPlansConfig.json';
-
+  private readonly SPENDING_PLANS_CONFIG: string | undefined =
+    typeof ConfigService.get('HBAR_SPENDING_PLANS_CONFIG') === 'string'
+      ? (ConfigService.get('HBAR_SPENDING_PLANS_CONFIG') as string)
+      : undefined;
```

Comment:
We should have `spendingPlansConfig.json` as a default value, and this should never be `undefined`.

- If the env variable is set:
  - It will always be parsed as a string: https://github.com/hashgraph/hedera-json-rpc-relay/blob/6442c98d95a9b8d6fcb12bb94c77e92fd67edfef/packages/config-service/src/services/globalConfig.ts#L326
- If the env variable is not set 
  - it will automatically resolve to the default value in `ValidationService#typeCasting` if we apply the suggestion from my previous comment: https://github.com/hashgraph/hedera-json-rpc-relay/blob/6442c98d95a9b8d6fcb12bb94c77e92fd67edfef/packages/config-service/src/services/validationService.ts#L53

Thus, we can replace this with:
```suggestion
  private readonly SPENDING_PLANS_CONFIG: string = ConfigService.get('HBAR_SPENDING_PLANS_CONFIG') as string;
```

Reply:
Replaced by simply loading it in the `loadSpendingPlansConfig()` and added the default value `spendingPlansConfig.json` to the `GlobalConfig`